YANG/tapi-photonic-media@2018-10-16.tree

```diff
@@ -183,3 +183,270 @@ module: tapi-photonic-media
              +--ro power-spectral-density?   decimal64
   augment /tapi-common:context/tapi-topology:topology-context/tapi-topology:topology/tapi-topology:node/tapi-topology:owned-node-edge-point/tapi-connectivity:cep-list/tapi-connectivity:connection-end-point:
     +--ro media-channel-assembly-spec
+  augment /tapi-common:get-service-interface-point-list/tapi-common:output/tapi-common:sip:
```

Comment:
I think the following is missing:
augment /tapi-common:context/tapi-common:service-interface-point:
    +--rw media-channel-service-interface-point-spec
          +--ro mc-pool

Reply:
It was already present in the previous version - line 65
[  augment /tapi-common:context/tapi-common:service-interface-point:](https://github.com/OpenNetworkingFoundation/TAPI/blob/0c566fe3b2c8efe925ddd5a7f9948f2df02a9b83/YANG/tapi-photonic-media%402018-10-16.tree#L64)

Reply:
Sorry I did not express well the point. My concern is why "get-service-interface-point-list" and "get-service-interface-point-details" are not augmented with:
+--rw media-channel-service-interface-point-spec
       +--ro mc-pool
          +--ro supportable-spectrum* []
          |  +--ro upper-frequency?        uint64
          |  +--ro lower-frequency?        uint64
          |  +--ro frequency-constraint
          |     +--ro adjustment-granularity?   adjustment-granularity
          |     +--ro grid-type?                grid-type
          +--ro available-spectrum* []
          |  +--ro upper-frequency?        uint64
          |  +--ro lower-frequency?        uint64
          |  +--ro frequency-constraint
          |     +--ro adjustment-granularity?   adjustment-granularity
          |     +--ro grid-type?                grid-type
          +--ro occupied-spectrum* []
             +--ro upper-frequency?        uint64
             +--ro lower-frequency?        uint64
             +--ro frequency-constraint
                +--ro adjustment-granularity?   adjustment-granularity
+--ro grid-type? grid-type

the same way the photonic model augment the service interface points data tree here:
[augment /tapi-common:context/tapi-common:service-interface-point:](https://github.com/OpenNetworkingFoundation/TAPI/blob/0c566fe3b2c8efe925ddd5a7f9948f2df02a9b83/YANG/tapi-photonic-media%402018-10-16.tree#L64)

Reply:
Thanks for the clarification - please review the commit #90bce6a below